New social icons

xentenza opened the pull request:
- added TikTok
- added Mastodon
- social-color.scss in alphabetical order

xentenza committed:
New social icons

- added TikTok
- added Mastodon
- social-color.scss in alphabetical order

alinekeller reviewed (changes requested):
Les couleurs définies dans le fichier social-colors.scss ne sont pas appliquées sur les icônes de Mastodon et Tiktok:

<img width="451" alt="" src="https://github.com/epfl-si/elements/assets/6631947/6d3a67b4-1b1d-4840-a308-e6ab3b1b6a75">

Il faut ajouter les styles suivants dans le fichier atoms/social-icon/social-icon.scss (vers la ligne 90, après les styles déjà définis pour les autres icônes):

```
.social-icon-mastodon {
  background-color: $mastodon;
}

.social-icon-tiktok {
  background-color: $tiktok;
}
```

xentenza committed:
PR 683 corrections

- corrections required by Aline;
- some misspellings...

williambelle commented on assets/components/atoms/social-icon/social-icon.scss:
```diff
@@ -80,6 +80,14 @@ a.social-icon {
+}
+
+.social-icon-tiktok {
+  background-color: $tiktok;
```
Le logo TikTok me semble minuscule comparé aux autres:
![Screenshot from 2023-10-03 14-44-23](https://github.com/epfl-si/elements/assets/2843501/7a39a749-80a7-40f5-b1db-077f184308a7)

On pourrait augmenter la taille de 1.3 comme YouTube ?

xentenza replied:
Ce sont les fichiers de base fournis par les TikTok Brand Guidelines. Mais bonne idée, c'est modifié dans ma nouvelle PR

xentenza committed:
TikTok svg size + alphabetical order

- Increased TikTok logo size *1.3
- social icons by alphabetical order
- TikTok and Mastodon not displayed in footer
